.github/workflows/doc-builder.yml

```diff
@@ -40,7 +40,16 @@ jobs:
         run: |
           cd ./src/AWS.Deploy.DocGenerator
           dotnet run --project ./AWS.Deploy.DocGenerator.csproj
+      - name: Check For Git Untracked Changes
+        id: gitcheck
+        shell: pwsh
+        run: |
+          $newFiles=$(git ls-files --others --exclude-standard | wc -l)
```

Comment:
Should we restrict the checks to only `.md` files?

Reply:
The following step only adds the doc site. No other changes will be committed.